src/Function, Proposition et Advanced Proposition Worlds.tex

```diff
@@ -0,0 +1,103 @@
+\documentclass[french,frenchkw]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{natbib}
+\usepackage{babel}
+\usepackage{graphicx}
+\usepackage{amsmath}
+\usepackage{tabto}
+\usepackage[T1]{fontenc}
+
+\graphicspath{{./Images/}}
+
+
+\title{Projet de Mathématiques : Assistant de Preuves}
+\author{Ivan Hasenohr (pour cette partie)}
+\date{Mars 2020}
+\begin{document}
+\maketitle
+\paragraph{Exact, Intro, Have, Apply}
+Ici nous allons présenter 4 techniques fondamentales pour l'utilisation de fonctions, une fonction en Lean étant exactement une fonction mathématique, donc un objet abstrait qui prend en argument un élément d'un ensemble A et renvoie un élément d'un ensemble B.\\
+\textbf{Exact}\\
+La première de ces tactiques est \textit{exact}. Elle permet de dire à Lean que le but recherché correspond exactement à ce que vous lui indiquez. Par exemple, si le but est un élément de P, et que vous disposez de $p \in P$, alors \textit{exact p,} terminera la preuve. De même, si le but est un élément de Q, et que vous disposez d'un élément p de P et d'une fonction f de P dans Q, alors \textit{exact f(p),} terminera la preuve.\\
```

Comment:
"si le but est un élément de P"
en réalité, c'est plutôt: si le but c'est "il existe un élément de type P", écris le en maths sans hésiter: $\exists p, p \in P$ ou quelque chose comme ça.